src/constants/common.ts

```diff
@@ -9,8 +12,12 @@ export const SEO_IMAGE =
 export const EMAIL_SUPPORT = "support@angelgiving.io";
 export const GENERIC_ERROR_MESSAGE = `An error occurred. Please get in touch with ${EMAIL_SUPPORT} if the problem persists.`;
 
-export const ENDOW_DESIGNATIONS = [
-  { label: "Charity", value: "Non-Profit", icon: "Charity" },
```

Comment:
@SovereignAndrey do these "Non/For-Profits" mentions also need to be renamed?
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Header/DesktopNav.tsx#L10
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Footer/constants.ts#L67
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Header/MobileNav/Menu/Links.tsx#L24
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/components/donation/Steps/Result/Success/Share.tsx#L65
https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/pages/Marketplace/Sidebar/Types.tsx#L7

Reply:
> @SovereignAndrey do these "Non/For-Profits" mentions also need to be renamed?
> 
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Header/DesktopNav.tsx#L10
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Footer/constants.ts#L67
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/App/Header/MobileNav/Menu/Links.tsx#L24
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/components/donation/Steps/Result/Success/Share.tsx#L65
> * https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/pages/Marketplace/Sidebar/Types.tsx#L7

No, those can all stay as is, as there we're speaking about all Non-profits on the platform (in general terms).